### ISSUE TITLE
PD Projector Screen Integration

### ISSUE THREAD

**Issue by TECHSAVYDUDE** [InZidiuZ/op-framework-issue-tracker#2619]
### Assurance

I have read the `README.md`

### Summary

     With the addition of “TV Carts” being able to play movies and YouTube videos; I think it would be a good add to be able to use the projector screens (the only ones I know of are in PDs) in a similar audio/visual display manner. 

     Where they would be different is that the projection screens would be able to load/present documents (PowerPoint, Google Docs, Google Sheets, Google Slides, etc.) With this you would be able to click back and forth and use the spaces as an actual meeting/training space.

     With this, however, the files would not remain in a “repository” for anyone in the server to access (like the TV Stands). This is because files may be personal or confidential and need to not be sharable to prevent Metagaming. 

### Reason

     Training has always been paramount for PD and all the divisions within. Currently files made have to be “emailed” in Discord and are flipped through freely in a no structured manner or not at all. With this change you would be able to have structured training and lead people uniformly through training, debriefs, operations, etc. without having to utilize another software simultaneously (Google, Discord, etc.)

     I believe it would also further the RP of departmental training by having players utilize those spaces in their full capacity.

**Comment by coalaura** [comment]
horrible formatting on god fr fr (i fixed it for you)

**Comment by TECHSAVYDUDE** [comment]
Yeah. I did it from my phone and it wouldn't let me create it within the app.

**Comment by InZidiuZ** [comment]
This has since been done. The libraries are now per-person & you can add raw image and video files into the UI.
